### DISCUSSION TITLE
Error: The security token included in the request is invalid for UAE region for aws-actions/configure-aws-credentials@v1

### DISCUSSION THREAD

**Discussion by akshayanilindrayani** [aws-actions/configure-aws-credentials#1132] [Q&A]
We have a reusable action for copying ecr image from dev "us-west-2" to other environments, and in that we create ecr repo on the target environment if it does not exists. For authentication we use the following block:

name: Configure Destination AWS Credentials
      uses: aws-actions/configure-aws-credentials@v1
      with:
        role-to-assume: arn:${{ inputs.destination-aws-partition }}:iam::${{ inputs.destination-aws-account-id }}:role/${{ inputs.destination-ecr-role }}
        aws-region: ${{ inputs.destination-aws-region }}

It works for all regions except UAE. On UAE, this block gives following error:

Run aws-actions/configure-aws-credentials@v1
  with:
    role-to-assume: arn:aws:iam::{account_id}:role/ECR_role
    aws-region: me-central-1
    audience: sts.amazonaws.com
  env:
    AWS_DEFAULT_REGION: us-west-2
    AWS_REGION: us-west-2
    AWS_ACCESS_KEY_ID: ***
    AWS_SECRET_ACCESS_KEY: ***
    AWS_SESSION_TOKEN: ***
Error: The security token included in the request is invalid

Is there any specific setting needed for UAE?

**Comment by kellertk** [comment]
v1 is very old, and does not use a version of the JS SDK that supports this region. Please use a more recent version of the action.
